tests/integration/api/test_api.py

```diff
@@ -553,6 +554,17 @@ def _compare_screenboard(board1, board2):
 
         share_res = dog.Screenboard.share(get_res['id'])
         assert share_res['board_id'] == get_res['id']
+        public_url = share_res['public_url'].replace('8080', '5000')
```

Comment:
The `replace('8080', '5000')` is a bit annoying, but I can't find a way to get rid of it in local env. Any idea @yannmh ? I guess it would work when testing against the staging env.